### DISCUSSION TITLE
Compatible with Android 11?

### DISCUSSION THREAD

**Discussion by tux9656** [schwabe/ics-openvpn#1323] [Q&A] [answered]
Is OpenVPN app compatable with Android 11? Are there any known issues with Android 11?

Thanks.

**Comment by schwabe** [comment, marked as answer]
There no known Android 11 specific issues.

**Reply by tux9656** [reply]
Thanks for the response. I just started getting the notification for the update to Android 11, so I’m going to take the plunge. I’ve been using this OpenVPN app for quite some time and it really is quite excellent. It’s actually my favorite implementation of OpenVPN of any platform. It’s very easy to set up and makes all the options and the configuration file very accessible.

**Reply by misanthropos** [reply]
I totally agree with @tux9656. Today however I tried the "Connect on boot" feature on Lineage 18.1 (Android 11). A Default VPN is set and working. But OpenVPN wont start on boot.
Not sure if that is related to Lineage or not.

**Reply by schwabe** [reply]
You should use the Android Setting "Always on VPN" instead. the on boot is no longer reliable working on modern Android versions.

**Reply by misanthropos** [reply]
Thanks! That indeed did the trick.

**Reply by zander-mazda** [reply]
Out of the topic sir , hope u could help me.

I'm trying to modify a vpn app using apk tool/editor. My source of knowledge about editing is from youtube therefore i am not good at or just a little bit of idea.

My main problem, there are parts that i could not be able to edit specific text/string/layout/xml. I dont know where to find that..

Sorry for my poor english. 

Attached is the sample that i wanted to edit.. In a pointed arrow..

Looking forward... Tnx
![AddText_07-08-11 39 47](https://user-images.githubusercontent.com/84930031/125019197-27fe3c80-e0a9-11eb-99fc-c88ee687d92e.jpg)
![AddText_07-08-11 48 13](https://user-images.githubusercontent.com/84930031/125019204-2a609680-e0a9-11eb-9404-e5adb788f627.jpg)
![AddText_07-08-11 31 21](https://user-images.githubusercontent.com/84930031/125019206-2b91c380-e0a9-11eb-8c1a-cd8b1205450a.jpg)
